npm/util/const.go

```diff
@@ -124,4 +124,24 @@ const (
 	AddNetworkPolicyEvent    string = "Add network policy"
 	UpdateNetworkPolicyEvent string = "Update network policy"
 	DeleteNetworkPolicyEvent string = "Delete network policy"
+
+	ErrorMetric  string = "ErrorMetric"
+	PackageName  string = "PackageName"
+	FunctionName string = "FunctionName"
+	ErrorCode    string = "ErrorCode"
+
+	BatchSize                 int = 32768
+	BatchInterval             int = 30
+	RefreshTimeout            int = 15
+	GetEnvRetryCount          int = 5
+	GetEnvRetryWaitTimeInSecs int = 3
+	AiInitializeRetryCount    int = 5
+	NpmID                     int = 1
+	IpsmID                    int = 2
+	IptmID                    int = 3
```

Comment:
Can we add a comment to some of these? It's hard to know if intervals or timeouts are in secs, mins. Some also seems a bit arbitrary (32768). Are the ids referenced from other places?

Reply:
For wrapping log and metrics, yes, I do think in same way to avoid duplicated logic.

Reply:
Sure

Reply:
![image](https://user-images.githubusercontent.com/16043262/92024122-96989e00-ed12-11ea-987e-65c4ed6203db.png)

Reply:
32768 came from this AI telemetry doc
 https://docs.microsoft.com/en-us/azure/azure-monitor/app/pricing
 
![image](https://user-images.githubusercontent.com/16043262/92059435-0ecc8700-ed46-11ea-9624-25c4905b4432.png)